story/ep1/pt/umi1_12.txt

```diff
@@ -1,7 +1,7 @@
 `O almoço terminou e os serviçais retiraram a mesa, e agora iam em direção à cozinha.`
 ` O vento e a chuva não estavam fortes, mas também não estavam fracos, e a ilha continuava incomunicável com o mundo exterior.`
 `Parecia que, primeiramente, Natsuhi havia planejado não deixar ninguém colocar um pé para fora daquela sala,`
-` mas quando chegou a hora de preparar o almoço, ela finalmente percebeu que era impossível.`
+` mas quando chegou a hora de preparar o almoço, ela finalmente percebeu que é impossível.`
```

Comment:
What is the purpose of changing the tense when the story is told in the past tense?

Reply:
Irei reverter.